app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -236,6 +237,14 @@ class __ChatInputState extends ConsumerState<_ChatInput> {
       baseOffset: cursorPos + emojiLength,
       extentOffset: cursorPos + emojiLength,
     );
+
+    // Ensure we keep the cursor up
+    // frame delay to keep focus connected with keyboard.
+    if (!chatFocus.hasFocus) {
+      Future.delayed(const Duration(milliseconds: 100), () {
```

Comment:
if we want this to happen for a "frame delay" shouldn't this be done in a `WidgetsBinding.instance.addPostFrameCallback((Duration duration) {` rather than a `Future.delayed`?!?

Reply:
Yeah I suppose it can be more sensible to use it! ;)